lib/WUI/wui.c

```diff
@@ -32,7 +32,8 @@ osMutexDef(wui_thread_mutex);   // Mutex object for exchanging WUI thread TCP th
 osMutexId(wui_thread_mutex_id); // Mutex ID
 
 static marlin_vars_t *wui_marlin_vars;
-wui_vars_t wui_vars; // global vriable for data relevant to WUI
+wui_vars_t wui_vars;                              // global vriable for data relevant to WUI
+static char wui_media_LFN[FILE_NAME_MAX_LEN + 1]; // static buffer for gcode file name
```

Comment:
Do you really need another buffer for the filename? marlinvars already have one...